### DISCUSSION TITLE
ondidichangecontent (event=>{})

### DISCUSSION THREAD

**Discussion by y912765390** [microsoft/monaco-editor#3636] [Q&A]
I want to know if the event '\t' can be counted as an alphabetic column instead of a standpoint. Such as tabsize = 4.

I want his column position to be 4 instead of 1. Is that OK

**Comment by hediet** [comment]
Sorry, can you elaborate on your question?

**Comment by y912765390** [comment]
I've been working on LSP recently and his ondidichange requires parameter calculations that are very similar to Monaco's didChangeContent but not quite the same especially in Monaco where the TAB button is a column and the JDT in Java needs to be calculated as the length of the tabsize Let's say I delete a TAB and I want to know if the event parameter of Monaco can be calculated as tabsize

**Comment by y912765390** [comment]
I want the character that the LSP needs but the event's column here is not going to be right if it's a TAB column minus one because she's treating the TAB as a character column instead of a tabsize character column. I wonder if I can change some parameter to make it return this event by the size of tabSize
